requirements.in

```diff
@@ -23,7 +23,7 @@ python-slugify
 requests
 social-auth-app-django
 wagtail-airtable
-wagtail==2.15.1
+wagtail==2.15.3
```

Comment:
patch version update for wagtail ensures that the "dependencies" entries don't lead to "missing migrations" errors